.github/scripts/update_changelog.py

```diff
@@ -17,36 +17,41 @@
 from datetime import datetime
 
 
-def get_commits_since_last_tag(current_version):
+def get_commits_since_last_tag(current_version, new_version):
     """
-    Get all commits since the last tag.
+    Get all commits between two tags using the GitHub API.
 
     Args:
-        current_version (str): The current version string
+        current_version (str): The starting version string
+        new_version (str): The ending version string
 
     Returns:
         list: A list of commit messages
     """
-    tag = f"v{current_version}"
+    start_tag = f"v{current_version}"
+    end_tag = f"v{new_version}"
 
-    # Check if tag exists
-    result = subprocess.run(["git", "tag", "-l", tag], capture_output=True, text=True)
+    range_spec = f"{start_tag}...{end_tag}"
```

Comment:
I do not think you need to update this - you can keep the previous range of `{tag}..HEAD` and it should be fine